lib/message.rb

```diff
@@ -0,0 +1,35 @@
+require 'twilio-ruby'
+# require 'dotenv'
+# require 'order'
+
+account_sid = '***'
+auth_token = '***'
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# from = '+447403930982' # Your Twilio number
```

Comment:
Think you could ultimately have had this as ENV["PHONE"] when uncommented and used somewhere